examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/smooth_quant/run_clm_no_trainer.py

```diff
@@ -233,32 +235,25 @@ def run_fn(model):
         else:
             acc = results["results"][task_name]["acc,none"]
     print("Accuracy: %.5f" % acc)
-    print('Batch size = %d' % args.batch_size)
+    print("Batch size = %d" % args.batch_size)
 
 if args.performance:
     user_model.eval()
-    from intel_extension_for_transformers.transformers.llm.evaluation.lm_eval import evaluate, LMEvalParser
+    batch_size, input_leng = args.batch_size, 512
```

Comment:
looks like we decide to change the design to skip using ITREX lm_eval and no reporting accuracy.  was it agreed in design review?

Reply:
It's a performance test so I don't think we need to use real data.
BTW, we have decided to re-write this example per Chang's request. This change will be delete and currently it's for our extension test. https://jira.devtools.intel.com/browse/ILITV-3682